### ISSUE TITLE
I have to patch the SDK to be able to work with relay

### ISSUE THREAD

**Issue by juan-voiq** [signalwire/signalwire-ruby-old#77]
I attach the files used to be able to use the relay_call.record and the amd! methods. Both relate to a conflict in the initialize initialize params.

```
Index: detect_action.rb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- detect_action.rb	(date 1580071947323)
+++ detect_action.rb	(date 1580071947323)
@@ -5,7 +5,7 @@
 module Signalwire::Relay::Calling
   class DetectAction < Action
     def result
-      DetectResult.new(@component)
+      DetectResult.new(component: @component)
     end
 
     def stop

```

```
Index: record_action.rb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- record_action.rb	(date 1580071947323)
+++ record_action.rb	(date 1580071947323)
@@ -5,7 +5,7 @@
 module Signalwire::Relay::Calling
   class RecordAction < Action
     def result
-      RecordResult.new(@component)
+      RecordResult.new(component: @component)
     end
 
     def stop

```
@lpradovera

**Comment by lpradovera** [comment]
Fixed in #78 , if you would like to test.

**Comment by juan-voiq** [comment]
Working for me
